hazelcast/src/main/java/com/hazelcast/monitor/impl/NodeStateImpl.java

```diff
@@ -30,11 +30,13 @@
 import java.util.List;
 import java.util.Map;
 
+import static com.hazelcast.util.JsonUtil.getLong;
 import static com.hazelcast.util.JsonUtil.getString;
 
 public class NodeStateImpl implements NodeState {
 
     private ClusterState clusterState;
+    private long clusterTime;
```

Comment:
There is already a `time` field in the `TimedMemberState` class. That field seems to be not initialized properly and always equal to `0L`, but it's sent to MC. I think it's worth it to use that field instead of introducing another one.

Alternatively this new `clusterTime` field should be kept and the old `TimedMemberState.time` field should be removed.

Reply:
Removal of that field is long overdue. Let me do that as part of this PR.

Reply:
Sounds great. Thanks.

Another minor thing to mention is that `clusterTime` might be a bit misleading, as it's local time on particular member. Shouldn't it be `memberTime`?